website/api/helpers/get-extended-osquery-schema.js

```diff
@@ -29,6 +36,31 @@ module.exports = {
     // Getting the specified osquery schema from the osquery/osquery-site GitHub repo.
     let rawOsqueryTables = await sails.helpers.http.get('https://raw.githubusercontent.com/osquery/osquery-site/source/src/data/osquery_schema_versions/'+VERSION_OF_OSQUERY_SCHEMA_TO_USE+'.json');
 
+    let rawOsqueryTablesLastModifiedAt;
+    if(includeLastModifiedAtValue) {
+      // If we're including a lastModifiedAt value for schema tables, we'll send a request to the GitHub API to get a timestamp of when the last commit
+      let responseData = await sails.helpers.http.get.with({
+        url: 'https://api.github.com/repos/osquery/osquery-site/commits',// [?]: https://docs.github.com/en/rest/commits/commits?apiVersion=2022-11-28#list-commits
+        data: {
+          path: '/src/data/osquery_schema_versions/'+VERSION_OF_OSQUERY_SCHEMA_TO_USE+'.json',
+          page: 1,
+          per_page: 1,//eslint-disable-line camelcase
+        },
+        headers: {
+          'User-Agent': 'fleet-schema-builder',
+          'Accept': 'application/vnd.github.v3+json',
+        },
+      }).intercept((err)=>{
+        return new Error(`When trying to send a request to GitHub get a timestamp of the last commit to the osqeury schema JSON, an error occurred. Full error: ${err}`);
```

Comment:
this will just look like `[Error error]` unless you `err.stack` or `util.inspect(err)` it